dotCMS/src/integration-test/java/com/dotmarketing/business/PermissionAPIIntegrationTest.java

```diff
@@ -226,28 +240,183 @@ public void issue11850() throws DotHibernateException, DotSecurityException, Dot
     			Permission inheritedPermission = new Permission(
     				Structure.class.getCanonicalName(), host.getPermissionId(), role.getId(), permission, true
     			);
-    			perm.save(inheritedPermission, host, sysuser, true);
+    			permissionAPI.save(inheritedPermission, host, systemUser, true);
 
     			// Check the permissions for content-type are now inherited from test host
-    			permissions = perm.getPermissions(type);
+    			permissions = permissionAPI.getPermissions(type);
     			assertFalse(permissions.isEmpty());
     			assertEquals(5, permissions.size());
 
     		} finally {
     			// Remove test content-type
-    			contentTypeApi.delete(type);
+    			contentTypeAPI.delete(type);
     		}
     	} finally {
     		// Remove test host
-    		try{
-    			HibernateUtil.startTransaction();
-    			APILocator.getHostAPI().archive(host, sysuser, false);
-    			APILocator.getHostAPI().delete(host, sysuser, false);
-    			HibernateUtil.commitTransaction();
-    		}catch(Exception e){
-    			HibernateUtil.rollbackTransaction();
-    			Logger.error(PermissionAPIIntegrationTest.class, e.getMessage());
-    		}
-    	}
+            removeTestHost(host);
+        }
+    }
+
+    /**
+     * https://github.com/dotCMS/core/issues/11962
+     */
+    @Test
+    public void test_permissionIndividuallyByRole() {
+
+        Host host = new Host();
+
+        Folder goQuestFolder = new Folder();
```

Comment:
just a warning, not a bug, not sure how I can modify it to make it work with the try/catch/finally logic that we need to clean stuff.